group-coordinator/src/main/resources/common/message/ConsumerGroupPartitionMetadataValue.json

```diff
@@ -29,7 +29,27 @@
       { "name": "TopicName", "versions": "0+", "type": "string",
         "about": "The topic name." },
       { "name": "NumPartitions", "versions": "0+", "type": "int32",
-        "about": "The number of partitions of the topic." }
+        "about": "The number of partitions of the topic." },
+      {
```

Comment:
Please, let's try to respect the existing format for the fields.

Reply:
Its the same? except the comma because I had to add an extra field

Reply:
Hum... It does not look the same, right?

For instance, existing fields looks like this

```
      { "name": "TopicName", "versions": "0+", "type": "string",
        "about": "The topic name." },
```

and the fields that you added look like this:

```
{
            "name": "Partition",
            "versions": "0+",
            "type": "int32",
            "about": "The partition number."
},
```

Reply:
Yes sorry, I was focused on the line above, I changed it now